chalicelib/model/Dynamo.py

```diff
@@ -20,6 +20,7 @@ class Meta(object):
     user_name = UnicodeAttribute()
     reason = UnicodeAttribute()
     hours = UnicodeAttribute()
+    lock = BooleanAttribute(default=False)
 
```

Comment:
this automatically creates a "lock" item as a Boolean which defaults to False when any new event is created.